dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java

```diff
@@ -1,102 +1,166 @@
+                .append("(select identifier, MIN(idate) as idate from contentlet, inode where inode.inode=contentlet.inode group by identifier) custom_select\n")
+                .append("where temp.identifier=custom_select.identifier\n")
+                .append("and inode.inode=temp.inode and inode.idate=custom_select.idate \n")
+                .append("and temp.structure_inode = struc.inode)\n")
+                .append("st ON (ident.id = st.myID) \n");
+
+        return query.toString();
+    }
+
+    @Override
+    public String getMSSQLScript() {
+        return getPostgresScript();
+    }
+
+    @Override
+    public String getH2Script() {
+        return null;
+    }
+
+    @Override
+    protected List<String> getTablesToDropConstraints() {
+        return null;
+    }
+
+    private String getQueryToUpdateNonContentletsOracle(final String tableName){
+        final StringBuilder query = new StringBuilder();
+        return query.append("MERGE INTO identifier ident\n")
+                .append("    USING\n")
+                .append("(select distinct temp.identifier myID, owner iowner, inode.idate idate from ")
+                .append(tableName).append(" temp, inode, \n")
+                .append("(select identifier, MIN(idate) as idate from ")
+                .append(tableName).append(", inode where inode.inode=")
+                .append(tableName).append(".inode group by identifier) custom_select\n")
+                .append("where temp.identifier=custom_select.identifier\n")
+                .append("and inode.inode=temp.inode and inode.idate=custom_select.idate)\n")
+                .append("st ON (ident.id = st.myID) \n")
+                .append("WHEN MATCHED THEN\n")
+                .append("    UPDATE SET ident.owner = st.iowner,\n")
+                .append("               ident.create_date = st.idate;\n").toString();
+    }
+
+    private String getQueryToUpdateNonContentletsPostgres(final String tableName){
+        final StringBuilder query = new StringBuilder();
+        return query.append("update identifier set owner=iowner, create_date=idate from\n")
+                .append("(select distinct temp.identifier myID, owner iowner, inode.idate idate from ")
+                .append(tableName).append(" temp, inode,\n")
+                .append("(select identifier, MIN(idate) idate from ")
+                .append(tableName).append(", inode \n")
+                .append("where inode.inode=").append(tableName).append(".inode group by identifier) custom_select  \n")
+                .append(" where temp.identifier=custom_select.identifier\n")
+                .append(" and inode.inode=temp.inode and inode.idate=custom_select.idate) my_query\n")
+                .append(" where  id=myID;\n").toString();
+    }
+
+    private String getQueryToUpdateNonContentletsMySQL(final String tableName){
+        final StringBuilder query = new StringBuilder();
+        return query.append("update identifier ident,\n")
```

Comment:
Codacy found an issue: [StringBuffer (or StringBuilder).append is called 2 consecutive times with literals. Use a single append with a single combined String.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6393742)